app/models/user.rb

```diff
@@ -20,4 +22,13 @@ def avatar
     "http://semantic-ui.com/images/avatar2/small/#{user}.png"
   end
 
+  def contributions
+    Edit.where(editor_id: id, state: "applied")
+  end
+
+  private
+
+    def hasherize_email
```

Comment:
I would suggest renaming the method `hash_email` or `hash_email_for_gravatar`, and change the attribute to `hashed_email` or `gravatar_id`.